### PR TITLE
French translation of the about and credit page

### PR DESCRIPTION
**Pull request by dje29**
Please have this reviewed by another French person before pulling.
French can sometimes be tricky, as well I always learned diving in English (sometimes in German) and may not be right with certain terminologies... So a second read is a **must**.
Finally I did not really know how to test rendering result as a website, it may be good to give it a try before actual deployment :)

### CODE REVIEW

**Comment by dirkhh** [comment]
I don't have a good test bed where to test the rendering, so the way I do this may not be a typical "production" environment - it goes straight to the live server.
I'll ask on the list for another french speaker to look at it.
THANKS

**Comment by dirkhh** [comment]
@Ghalr - would you be able to take a look?

File: _pages/about.fr [hunk 1]
```diff
@@ -34,40 +34,39 @@ Enregistrez vos données de carnet de plongée dans le cloud. Travaillez avec vo
...
 
-Subsurface can <strong>track</strong> single- and multi-tank <strong>dives</strong> using air, Nitrox or TriMix. It allows tracking of dive locations including GPS coordinates (which can also conveniently be entered using a map interface), logging of equipment used and names of other divers, and lets users rate dives and provide additional notes. 
+Subsurface permet le <strong>suivi</strong> de <strong>plongées</strong> a l'air, au Nitrox ou au TriMix pour une ou plusieurs bouteilles. Il donne la possibilité du suivit des sites de plongées ainsi que leur position GPS (cette dernière peut être facilement saisie a l'aide d'une carte), de l'enregistrement de l'équipement utilisé, du nom des autres plongeurs, et de remarques supplémentaires ainsi que d'une notation des plongées.
```
Comment (pypingou) [inline _pages/about.fr]: Il donne la possibilité d'enregistrer les sites de plongées..., l'équipement utilisé, le nom des autres plongeurs, noté les plongées et information supplémentaire désirée?

File: _pages/about.fr [hunk 2]
```diff
@@ -34,40 +34,39 @@ Enregistrez vos données de carnet de plongée dans le cloud. Travaillez avec vo
...
 
-Users can tag dives and <strong>filter a dive list</strong> based on criteria including tags, locations and people with whom you were diving.
+Les utilisateurs peuvent taguer les plongées et <strong>filtrer la liste de plongées</strong> a l'aide de différents critères incluant les tags, les lieux et personnes avec qui vous avez plongé.
```
Comment (pypingou) [inline _pages/about.fr]: s/taguer/étiqueter/? (du coup faudrait revoir aussi le: critères incluant ces étiquettes, les lieux...

File: _pages/about.fr [hunk 3]
```diff
@@ -34,40 +34,39 @@ Enregistrez vos données de carnet de plongée dans le cloud. Travaillez avec vo
...
 
 [/et_pb_text][et_pb_image admin_label="Image" src="https://subsurface-divelog.org/wp-content/uploads/2011/10/TagsAndRating1.png" alt="Screenshot of rating, tags and other data fields" show_in_lightbox="off" url_new_window="off" animation="left" sticky="off" align="center" force_fullwidth="off" always_center_on_mobile="on" use_border_color="on" border_color="#878787" border_style="solid" custom_margin="1em|1em|2em|1em" saved_tabs="all"]
 
 
 
 [/et_pb_image][et_pb_text admin_label="Text" background_layout="light" text_orientation="left" use_border_color="off" border_color="#ffffff" border_style="solid"]
 
-Subsurface also calculates a wide variety of <strong>statistics</strong> of the user’s diving and tracks information like the SAC rate, partial pressures of O2, N2 and He, calculated deco information, and many more.
+Subsurface calcule également un grand nombre de <strong>statistiques</strong> de plongée de l'utilisateur. Il permet le suivit de la consommation d'air (SAC), de la pression partielle d'O2, de N2 et d'He, des calculs d'information de déco, ainsi que beaucoup d'autres valeurs.
```
Comment (pypingou) [inline _pages/about.fr]: s/de plongée de l'utilisateur./sur les plongées de l'utilisateur./

File: _pages/about.fr [hunk 4]
```diff
@@ -34,40 +34,39 @@ Enregistrez vos données de carnet de plongée dans le cloud. Travaillez avec vo
...
 
-The program is localized in about <strong>20 languages</strong> and well supported by an active developer community.
+Le programme est traduit en à peu près <strong>20 langues</strong> et est très bien supporté par une communauté active de développeurs.
```
Comment (pypingou) [inline _pages/about.fr]: supporté -> maintenu

File: _pages/about.fr [hunk 5]
```diff
@@ -34,40 +34,39 @@ Enregistrez vos données de carnet de plongée dans le cloud. Travaillez avec vo
...
 
 [/et_pb_text][/et_pb_column][et_pb_column type="1_2"][et_pb_text admin_label="Text" background_layout="light" text_orientation="left" use_border_color="off" border_color="#ffffff" border_style="solid"]
 
-One of the major strengths of Subsurface is its support of a wide range of <strong>dive computers</strong> (see the list of supported dive computers). Subsurface can also import existing dive logs from several sources including MacDive, Suunto DM3, DM4 &amp; DM5, JDiveLog and divelogs.de. 
+Une des importances majeures de Subsurface est son support pour un très grand nombre d'<strong>ordinateurs de plongée</strong> (voire la liste d'ordinateurs supporté). Subsurface peut également importer les logs de plongée de plusieurs sources tels que MacDive, Suunto DM3, DM4 &amp; DM5, JDiveLog et divelogs.de.
```
Comment (pypingou) [inline _pages/about.fr]: Support c'est un terme anglais, du coup: sa maintenance, ses mises à jour?

File: _pages/about.fr [hunk 6]
```diff
@@ -34,40 +34,39 @@ Enregistrez vos données de carnet de plongée dans le cloud. Travaillez avec vo
...
 
-Another strength is its ability to <strong>visualize</strong> the depth profile (and, if available, the tank pressure curve) in innovative ways that give the user additional information on relative velocity, and momentary air consumption, during a dive.
+Un autre atout est ça capacité a <strong>visualiser</strong> le profil de profondeur (et, si disponible, la courbe de pression des bouteilles) de manière innovante, qui donne à l'utilisateur les informations de vitesse relative, et la consomation d'air momentané pendant une plongée.
```
Comment (pypingou) [inline _pages/about.fr]: est ça -> est sa ?

File: _pages/credits.fr [hunk 7]
```diff
@@ -9,33 +9,33 @@ permalink: https://subsurface-divelog.org/credits/
 published: true
 post_language: fr
 ---
-Subsurface is itself open source software released under the <a href="https://github.com/Subsurface-divelog/subsurface/blob/master/LICENSE">GPLv2</a> - and it is based on many pieces of open source software.
+Subsurface est en lui-même un logiciel libre délivré sous licence <a href="https://github.com/Subsurface-divelog/subsurface/blob/master/LICENSE">GPLv2</a> - et est basé sur plusieurs éléments de logiciel libre.
```
Comment (pypingou) [inline _pages/credits.fr]: est utilise d'autres logiciels libre?

File: _pages/about.fr [hunk 8]
```diff
@@ -34,40 +34,39 @@ Enregistrez vos données de carnet de plongée dans le cloud. Travaillez avec vo
...
+A l'automne 2011, lorsque qu'une accalmie forcée dans le développement du noyau lui donna la possibilité de démarrer un nouveau projet, le créateur de Linux, Linus Torvalds, décida de s'attaquer à sa frustration causée par le manque d'un programme de logs de plongée décent supportant Linux. Linus travailla avec une équipe de développeurs, et Subsurface en est le résultat. Il supporte actuellement Linux, Windows et MacOS. Il permet l'importation depuis un grand nombre d'ordinateurs de plongée ainsi que plusieurs programmes de logs de plongée existants. Il donne un accès facile et rapide à la visualisation des informations fournies par les ordinateurs de plongée modernes et permet aux utilisateurs de suivre un grand nombre de données au sujet de leurs plongées. En automne 2012 Dirk Hohndel a pris en charge le rôle de Mainteneur de Subsurface.
```
Comment (pypingou) [inline _pages/about.fr]: logs de plongées ou journal de plongées?

Mais dans ce cas il faudra harmoniser partout log -> journal

Comment (pypingou) [inline _pages/about.fr]: il permet l'importation -> il permet l'import

**Comment by pypingou** [comment]
Basically, I think this would be understandable by a french person as is, but some of the formulation used are closer to english than to what we would use in native french, so reformulating them would improve them.

There are a few questions as well: should we translate ``log`` to ``journal``? (The proper French term) Same thing for ``tags`` to ``étiquettes`` (for both of them the question is: are these words that is sufficiently known to french person to leave them in english or should we translate them all the time?)

**Comment by dirkhh** [comment]
It makes me very happy to see a discussion that I cannot follow. Thanks, @pypingou !
@dje29 - for this repo it's fine if you just want to add another commit that updates the text, no reason to redo the pull request, just add to it. On the subsurface repository I usually like to keep things reasonably clean, but since this is just a tool to feed the WordPress site, I really don't care all that much about the number of commits...

**Comment by pypingou** [comment]
Tout le plaisir est pour moi @dirkhh :)

**Comment by dirkhh** [comment]
Google translate suggests I might respond with... "Votre aide est très appréciée"
How badly did it do? :-)

**Comment by pypingou** [comment]
Not too bad, not too bad :)

**Comment by Ghalr** [comment]
I do not know if the French manual were translated with Google Translation but some sentences are kinda not translated by a French.

It can be interesting also to review the whole French language, as I am totally agree with @pypingou 's examples about log->journal and tags->étiquettes. French users do understand those terms. But as Dive log is more shorter than "Journal de plongée" this will depend if we got enought place to display the french translation.

I can have a look later today.

File: _pages/about.fr (hunk 1 above)
Comment (dje29) [reply _pages/about.fr]: I inspired myself from your proposal, but found a formulation that seemed slightly better.

File: _pages/about.fr (hunk 6 above)
Comment (dje29) [reply _pages/about.fr]: Good one... when I said that French can be tricky :D

**Comment by dje29** [comment]
Hi guys,

Here is a first round of updates based on your proposals. (I did amend to avoid mess in git history )

I had a talk with some friends working for France plongée, they did confirm the use of "Journal de plongées", as for tag(s) I browsed around and in several places the term étiquettes was indeed used to translate tags. (e.g. Exel's data tags  becomes étiquettes de données on msdn)

Thanks for the review :)

File: _pages/about.fr [hunk 9]
```diff
@@ -34,40 +34,39 @@ Enregistrez vos données de carnet de plongée dans le cloud. Travaillez avec vo
...
 
-Subsurface can <strong>track</strong> single- and multi-tank <strong>dives</strong> using air, Nitrox or TriMix. It allows tracking of dive locations including GPS coordinates (which can also conveniently be entered using a map interface), logging of equipment used and names of other divers, and lets users rate dives and provide additional notes. 
+Subsurface permet le <strong>suivi</strong> de <strong>plongées</strong> a l'air, au Nitrox ou au TriMix pour une ou plusieurs bouteilles. Il donne la possibilité d'enregistrer les sites de plongées ainsi que leur position GPS (cette dernière peut être facilement saisie a l'aide d'une carte), l'équipement utilisé, le nom des autres plongeurs, des informations supplémentaires ainsi que de noter les plongées.
```
Comment (pypingou) [inline _pages/about.fr]: peut être facilement saisie -> peut aussi être facilement saisie?

**Comment by pypingou** [comment]
One small suggestion otherwise looks good to me :)

**Comment by dje29** [comment]
Last suggestion from @pypingou included, thanks for reviewing.

**Comment by dirkhh** [comment]
Thanks for working on this - really appreciated.
Now I need to convince WordPress to listen to the push request from GitHub. I have problems with that over the weekend, but I'm hopeful that this has now been fixed.

**Comment by dje29** [comment]
Seems it works :)

File: _pages/about.fr [hunk 10]
```diff
@@ -34,40 +34,39 @@ Enregistrez vos données de carnet de plongée dans le cloud. Travaillez avec vo
...
 
 [/et_pb_text][et_pb_image admin_label="Image" src="https://subsurface-divelog.org/wp-content/uploads/2011/10/TagsAndRating1.png" alt="Screenshot of rating, tags and other data fields" show_in_lightbox="off" url_new_window="off" animation="left" sticky="off" align="center" force_fullwidth="off" always_center_on_mobile="on" use_border_color="on" border_color="#878787" border_style="solid" custom_margin="1em|1em|2em|1em" saved_tabs="all"]
 
 
 
 [/et_pb_image][et_pb_text admin_label="Text" background_layout="light" text_orientation="left" use_border_color="off" border_color="#ffffff" border_style="solid"]
 
-Subsurface also calculates a wide variety of <strong>statistics</strong> of the user’s diving and tracks information like the SAC rate, partial pressures of O2, N2 and He, calculated deco information, and many more.
+Subsurface calcule également un grand nombre de <strong>statistiques</strong> sur les plongées de l'utilisateur. Il permet le suivit de la consommation d'air (SAC), de la pression partielle d'O2, de N2 et d'He, des calculs d'information de déco, ainsi que beaucoup d'autres valeurs.
```
Comment (alexandrebelloni) [inline _pages/about.fr:47]: Subsurface calcule également un grand nombre de <strong>statistiques</strong> sur les plongées de l'utilisateur. Il permet le suivi de la consommation d'air (SAC), de la pression partielle d'O2, de N2 et d'He, des calculs d'informations de déco, ainsi que beaucoup d'autres valeurs.

File: _pages/about.fr [hunk 11]
```diff
@@ -34,40 +34,39 @@ Enregistrez vos données de carnet de plongée dans le cloud. Travaillez avec vo
...
 
-One of the major strengths of Subsurface is its support of a wide range of <strong>dive computers</strong> (see the list of supported dive computers). Subsurface can also import existing dive logs from several sources including MacDive, Suunto DM3, DM4 &amp; DM5, JDiveLog and divelogs.de. 
+Une des importances majeures de Subsurface est sa compatibilité avec un très grand nombre d'<strong>ordinateurs de plongée</strong> (voire la liste d'ordinateurs pris en charge). Subsurface peut également importer le journal de plongées de plusieurs sources tels que MacDive, Suunto DM3, DM4 &amp; DM5, JDiveLog et divelogs.de.
 
-Another strength is its ability to <strong>visualize</strong> the depth profile (and, if available, the tank pressure curve) in innovative ways that give the user additional information on relative velocity, and momentary air consumption, during a dive.
+Un autre atout est sa capacité a <strong>visualiser</strong> le profil de profondeur (et, si disponible, la courbe de pression des bouteilles) de manière innovante, qui donne à l'utilisateur les informations de vitesse relative, et la consomation d'air momentané pendant une plongée.
```
Comment (alexandrebelloni) [inline _pages/about.fr:55]: should be: "momentanée"

**Review by alexandrebelloni — COMMENTED** [review]
Few typos

**Pull request merged**